README.md

```diff
@@ -1,4 +1,5 @@
-[![Python 3.7 3.9](https://img.shields.io/badge/python-3.7%20%7C%203.9-blue.svg)](https://www.python.org/)
+[![VFX Platform](https://img.shields.io/badge/vfxplatform-2024%20%7C%202023%20%7C%202022%20%7C%202021-blue.svg)](http://www.vfxplatform.com/)
```

Comment:
Please remove 2024. Python 3.10 is VFX 2023

Reply:
Ah good catch, I missed that.